docs/source/gbd2019_models/causes/ischemic_stroke/index.rst

```diff
@@ -311,365 +342,34 @@ State and Transition Data Tables
-     - Source 
-     - Sink 
-     - Value
-     - Notes
-   * - i
-     - S
-     - I
-     - 
-     - 
-   * - r
-     - I
-     - R
-     - 	
-     - 
- 
-
-An example, if the data is present in GBD,
-
-.. list-table:: Transition Data
-   :widths: 10 10 10 20 30
-   :header-rows: 1
-   
-   * - Transition
-     - Source 
-     - Sink 
-     - Value
-     - Notes
-   * - i
-     - S
-     - I
-     - :math:`\frac{\text{incidence_rate_cid}}{\text{1 - prevalence_cid}}`
-     - 
-   * - r
-     - I
-     - R
-     - remission_rate_cid
-     - 
-
-Sometimes, we might need to use *modelable entity id* to get data. Sometimes, we might need to calculate remission rate 
-based on average case duration. In that case, the row would look like,
-
-.. list-table:: Transition Data
-   :widths: 10 10 10 20 30
-   :header-rows: 1
-   
-   * - Transition
-     - Source 
-     - Sink 
-     - Value
-     - Notes
-   * - r
-     - I
-     - R
-     - remission_rate_cid :math:`= \frac{\text{365 person-days}}{\text{average case duration in days} \times \text{1 year}}`
-     - 
-	 
-
-Data Sources
-""""""""""""
-
-This table contains the data sources for all the measures. The table structure and common measures are as below:
-
-.. list-table:: Data Sources
-   :widths: 20 25 25 25
-   :header-rows: 1
-   
-   * - Measure
-     - Sources
-     - Description
-     - Notes
-   * - prevalence_cid
-     - 
-     - 
-     - 
-   * - birth_prevalence_cid
-     - 
-     - 
-     -
-   * - deaths_cid
-     - 
-     - 
-     - 
-   * - population
-     - 
-     - 
-     - 
-   * - sequelae_cid
-     - 
-     - 
-     - 
-   * - incidence_rate_cid
-     - 
-     - 
-     - 
-   * - remission_rate_m1594
-     - 
-     - 
-     - 
-   * - disability_weight_s{`sid`}
-     - 
-     - 
-     - 
-   * - prevalence_s{`sid`}
-     - 
-     - 
-     - 
-
-An example, that contains common sources for the measures,
-
-.. list-table:: Data Sources
-   :widths: 20 25 25 25
-   :header-rows: 1
-   
-   * - Measure
-     - Sources
-     - Description
-     - Notes
-   * - prevalence_cid
-     - como
-     - Prevalence of cause
-     - 
-   * - birth_prevalence_cid
-     - como
-     - Birth prevalence of cause
-     -
-   * - deaths_cid
-     - codcorrect
-     - Deaths from cause
-     - 
-   * - population
-     - demography
-     - Mid-year population for given age/sex/year/location
-     - 
-   * - sequelae_cid
-     - gbd_mapping
-     - List of sequelae
-     - 
-   * - incidence_rate_cid/mid
-     - como/dismod
-     - Incidence rate for cause
-     - 
-   * - remission_rate_cid/mid
-     - como/dismod
-     - Remission rate for cause
-     - 
-   * - disability_weight_s{`sid`}
-     - YLD appendix
-     - Disability weight of sequela with id `sid`
-     - 
-   * - prevalence_s{`sid`}
-     - como
-     - Prevalence of sequela with id `sid`
-     - 
 
 
 Validation Criteria
 +++++++++++++++++++
 
+1. Compare CSMR experienced by simulants to CSMR from CoDCorrect in GBD.
```

Comment:
This makes me understand more about the CSMRs---do you plan to validate against total, or against D_A and D_C as well?

Reply:
@catherineojohnson can you weigh in on this question?

Reply:
I am going to merge this branch with the CSMRs and can update in the future after more discussion, or if there is confusion from the SEs.